Docs: mention `docsify` on "Build customization" (#9439)
readthedocs/readthedocs.org: docs/user/build-customization.rst

and :ref:`extending the build process <build-customization:extend the build process>` is not enough,
all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands` configuration file key.

As Read the Docs does not have control over the build process,
you are responsible for running all the commands required to install requirements and build your project properly.

> Once the build process finishes, the content of the ``_readthedocs/html/`` directory will be hosted.

Reviewer: ```suggestion
Once the build process finishes, the contents of the ``_readthedocs/html/`` directory will be hosted.
```

Examples
++++++++

> This section contains some examples that can be seen as showcase of what it's possible to do with :ref:`config-file/v2:build.commands`.

Reviewer: Some reduction for clarity

```suggestion
This section contains some examples that showcase what is possible with :ref:`config-file/v2:build.commands`.
```

Author: I force myself to reduce verbosity while writing, but it seems that being Argentinian always wins here 😂

> Note that you may need to modify and adapt them depending your needs.

Reviewer: ```suggestion
Note that you may need to modify and adapt these examples depending on your needs.
```

       - pelican --settings docs/pelicanconf.py --output _readthedocs/html/ docs/

Docsify
~~~~~~~

> `Docsify <https://docsify.js.org/>`_ generates documentation websites on the fly, without the need of building static HTML.

Reviewer: ```suggestion
`Docsify <https://docsify.js.org/>`_ generates documentation websites on the fly, without the need to build static HTML.
```
